specs/experimental/contracts/L1/l1-cross-domain-messenger.md

```diff
@@ -179,7 +167,7 @@ Sends a message to a target address on L2 by depositing a transaction through th
 - MUST call portal.depositTransaction with the encoded message, base gas, and msg.value
 - MUST emit SentMessage event with target, sender, message, nonce, and gas limit
 - MUST emit SentMessageExtension1 event with sender and value
-- MUST increment the message nonce by one
+- MUST increment the [message nonce](../L2/l2-to-l1-message-passer.md#message-nonce) by one
```

Comment:
devinai Message nonce used in the CrossDomainMessenger is not the same as as the one in the L2ToL1MessagePasser.